x-pack/plugins/upgrade_assistant/public/application/components/overview/migrate_system_indices/migrate_system_indices.tsx

```diff
@@ -123,10 +124,55 @@ const i18nTexts = {
   },
 };
 
+const ConfirmModal: React.FC<{
+  onCancel: () => void;
+  onConfirm: () => void;
+}> = ({ onCancel, onConfirm }) => (
+  <EuiConfirmModal
+    title={i18n.translate('xpack.upgradeAssistant.overview.systemIndices.confirmModal.title', {
+      defaultMessage: 'Migrate Indices',
+    })}
+    onCancel={onCancel}
+    onConfirm={onConfirm}
+    cancelButtonText={i18n.translate(
+      'xpack.upgradeAssistant.overview.systemIndices.confirmModal.cancelButton.label',
+      {
+        defaultMessage: 'Cancel',
+      }
+    )}
+    confirmButtonText={i18n.translate(
+      'xpack.upgradeAssistant.overview.systemIndices.confirmModa.confirmButton.label',
+      {
+        defaultMessage: 'Confirm',
+      }
+    )}
+    defaultFocusedButton="confirm"
+  >
+    {i18n.translate('xpack.upgradeAssistant.overview.systemIndices.confirmModa.description', {
+      defaultMessage:
+        'Migrating system indices may lead to downtime while the indices are reindexed. Are you sure you want to migrate system indices?',
```

Comment:
(Reviewing instead of @florent-leborgne while he's on paternity leave.) Thanks for reaching out @sabarasaba!
I think we could simplify it a little and remove the "Are you sure" statement, and then instead of "Confirm" in the button, we could say "Migrate anyway". What do you think?

Reply:
Love it, thanks a lot for chiming in!